hits_cpp/hits.cpp

```diff
@@ -212,6 +212,16 @@ MemeRatingMap getRatings(Memes& memes, MemeShufflesMap& shuffles, bool verbose,
     return ratings;
 }
 
+void setHitsFactor(float factor, Memes& memes){
```

Comment:
`factorHits` would be a better name.
Also for better convention, swap the arguments place, usually first argument is the data/vector and second is the scalar